plugin/multilevelProposeROI/multilevelProposeROIPlugin.cpp

```diff
@@ -405,7 +405,7 @@ int MultilevelProposeROI::enqueue(
             static_cast<float>(mImageSize.d[1]), // Input Height
             static_cast<float>(mImageSize.d[2]),
             DataType::kFLOAT, // mType,
-            mParam, proposal_ws, workspace + kernel_workspace_offset,
+            mParam, proposal_ws, reinterpret_cast<uint8_t*>(workspace) + kernel_workspace_offset,
```

Comment:
Please update to `static_cast` to go from `void *` to `T *`:
https://github.com/NVIDIA/TensorRT/blob/master/CODING-GUIDELINES.md#casts